tests/unit/planning/test_layer_planner.py

```
@@ -0,0 +1,288 @@
+
+
+@pytest.fixture
+def four_drones() -> list[DroneStateView]:
+    return _drones([(0, 1.0), (1, 1.0), (2, 1.0), (3, 1.0)])
+
+
+# --------------------------------------------------------------------------- #
+# _largest_remainder                                                          #
+# --------------------------------------------------------------------------- #
+@pytest.mark.parametrize(
+    "n, weights",
```

Ruff (0.16.3)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)